.deny.toml

```
@@ -26,3 +26,6 @@ allow = [
 unknown-registry = "deny"
 unknown-git = "deny"
 required-git-spec = "rev"
+
+[advisories]
+ignore = ["RUSTSEC-2025-0023"]
```

```suggestion
ignore = ["RUSTSEC-2025-0023"] // FIXME: Remove this once Firefox moves to libc >= XX.YY
```
Where `XX.YY` is whatever is correct.